src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java

```diff
@@ -0,0 +1,137 @@
+        CARDS.add(true);
+        CARDS.add(true);
+    }
+
+    private static void generationData() throws SQLException, NamingException {
+        connection = ConnectionPool.getConnection();
+        int rand = (int) (Math.random() * USERS.size()) + 0;
+
+        String[] userArray;
+        userArray = (USERS.remove(rand)).split(",");
+        String loginUser = userArray[0];
+        String passwordUser = userArray[1];
+        int idRoleUser = Integer.parseInt(userArray[3]);
+        String nameUser = userArray[4];
+
+        PreparedStatement psUsers = connection.prepareStatement(SQL_USERS);
+        psUsers.setString(1, loginUser);
+        psUsers.setString(2, passwordUser);
+        psUsers.setInt(3, idRoleUser);
+        psUsers.setString(4, nameUser);
+        psUsers.executeUpdate();
+
+        String[] accountArray;
+        accountArray = ACCOUNT.remove(rand).split(",");
+        PreparedStatement psIdUser = connection.prepareStatement(SQL_ID_USER);
+        psIdUser.setString(1, loginUser);
+
+        ResultSet rsUsers = psIdUser.executeQuery();
+        rsUsers.next();
+        int idUsers = rsUsers.getInt(1);
+        int amountAccounts = Integer.parseInt(accountArray[0]);
+        boolean isActiveAccounts = Boolean.parseBoolean(accountArray[1]);
+
+        PreparedStatement psAccount = connection.prepareStatement(SQL_ACCOUNTS);
+        psAccount.setInt(1, idUsers);
+        psAccount.setInt(2, amountAccounts);
+        psAccount.setBoolean(3, isActiveAccounts);
+        psAccount.executeUpdate();
+
+        PreparedStatement psIdAccount = connection.prepareStatement(SQL_ID_ACCOUNT);
+        psIdAccount.setInt(1, idUsers);
+
+        ResultSet rsAccount = psIdAccount.executeQuery();
+        rsAccount.next();
+        int idAccount = rsAccount.getInt(1);
+        boolean isActive = CARDS.remove(rand);
+
+        PreparedStatement psCards = connection.prepareStatement(SQL_CARDS);
+        psCards.setInt(1, idUsers);
+        psCards.setInt(2, idAccount);
+        psCards.setBoolean(3, isActive);
+        psCards.executeUpdate();
+
+        rsAccount.close();
+        rsUsers.close();
+        ConnectionPool.connectionRelease(connection);
+    }
+
+    private static boolean checkedRoleAdmin() throws SQLException, NamingException {
+        connection = ConnectionPool.getConnection();
+        int idRoleAdmin = 1;
+
+        PreparedStatement psIdRoleUser = connection.prepareStatement(SQL_ID_ROLE_USER);
+        psIdRoleUser.setInt(1, idRoleAdmin);
+        ResultSet rsIdRoleUser = psIdRoleUser.executeQuery();
+
+        if (rsIdRoleUser.next()) {
+            return true;
+        }
+        rsIdRoleUser.close();
+        ConnectionPool.connectionRelease(connection);
+        return false;
+    }
+
+    public static boolean addTestData() {
+        try {
+            try {
+                if(checkedRoleAdmin()){
+                    return true;
+                }
+
+                for (int i = 0; i < DATA_COUNT; i++) {
```

Comment:
Do we really need this empty loop?

Reply:
This is my mistake, there should be a method: generationData()